src/main/java/net/sf/jabref/model/entry/BibLatexEntryTypes.java

```diff
@@ -871,4 +871,10 @@ public String getName() {
     public static Optional<EntryType> getType(String name) {
         return ALL.stream().filter(e -> e.getName().equalsIgnoreCase(name)).findFirst();
     }
+
+    public static final List<String> BIBLATEX_PERSON_NAME_FIELDS = Arrays.asList("author", "editor", "editora",
```

Comment:
Are not all kinds of BibTex fields already  somewhere defined  globally as
constants?
e.g. author, editor. Then it would be easy to just use them,. e.g
GlobalFields.AUTHOR or somehow...

And is "editora" a correct field?


2016-04-04 22:46 GMT+02:00 Tobias Diez <notifications@github.com>:

> In src/main/java/net/sf/jabref/model/entry/BibLatexEntryTypes.java
> <https://github.com/JabRef/jabref/pull/1097#discussion_r58445744>:
>
> > @@ -871,4 +871,10 @@ public String getName() {
> >      public static Optional<EntryType> getType(String name) {
> >          return ALL.stream().filter(e -> e.getName().equalsIgnoreCase(name)).findFirst();
> >      }
> > +
> > +    public static final List<String> BIBLATEX_PERSON_NAME_FIELDS = Arrays.asList("author", "editor", "editora",
>
> I would move these lists to InternalBibtexFields.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/JabRef/jabref/pull/1097/files/4fabed3c696dbdb33c4283183705ffecccd645a8#r58445744>
>